core/src/scss/components/accordion/_accordion.scss

```diff
@@ -9,7 +9,98 @@
 //
 // Style guide: Simple.Accordion
 //
+
 .su-accordion {
+  @include font-smoothing;
+}
+
+.su-accordion,
+.su-accordion__list,
+.su-accordion__item,
+.su-accordion__title,
+.su-accordion__content {
   clear: both;
   position: relative;
 }
+
+.su-accordion__collapse-all,
+.su-accordion__expand-all {
+  float: right;
+}
+
+.su-accordion__collapse-all {
+  @include modular-spacing('margin', null null 2);
+  @include margin(null null null 1.2em);
+}
+
+.su-accordion__list {
+  @include list-unstyled;
+  border-top: 1px solid $su-color-cool-grey-25;
+}
+
+.su-accordion__item {
+  border-bottom: 1px solid $su-color-cool-grey-25;
+}
+
+.su-accordion__title {
+  @include modular-typography(1);
+  @include margin(0);
+  width: 100%;
+
+  > .su-accordion__button {
+    all: inherit;
```

Comment:
This property is amazing. TIL.
Possible IE issue tho: https://caniuse.com/#feat=css-all

Reply:
I just learned too. And see above.